sys/fmt/fmt.c

```diff
@@ -118,6 +123,25 @@ size_t fmt_s32_dec(char *out, int32_t val)
     return fmt_u32_dec(out, val) + negative;
 }
 
+size_t fmt_s16_dec(char *out, int16_t val)
```

Comment:
is this function really necessary if it's just a wrapper around `fmt_s32_dec`? The user could call `fmt_s32_dec` directly.
If you insist on keeping `fmt_s16_dec` we could at least inline it?